src/test/kotlin/org/rust/ide/inspections/lints/RsRedundantSemicolonsInspectionTest.kt

```diff
@@ -0,0 +1,72 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections.lints
+
+import org.rust.ide.inspections.RsInspectionsTestBase
+
+class RsRedundantSemicolonsInspectionTest : RsInspectionsTestBase(RsRedundantSemicolonsInspection::class) {
```

Comment:
Actually, I'd add the following tests:
- with more common code than local items. Like `let a = 1 + 1;;`. Of course, it's also just an empty statement, but it seems more actual case
- with `allow(redundant_semicolons)` and `allow(unused)` attributes
- with suppress quick-fix